spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -141,15 +141,28 @@
     before do
       n = 2
       @pm = PolicyMachine.new(name: 'ActiveRecord PM', storage_adapter: PolicyMachineStorageAdapter::ActiveRecord)
+      @pm2 = PolicyMachine.new(name: '2nd ActiveRecord PM', storage_adapter: PolicyMachineStorageAdapter::ActiveRecord)
+      @pm3 = PolicyMachine.new(name: '3rd ActiveRecord PM', storage_adapter: PolicyMachineStorageAdapter::ActiveRecord)
+
       @u1 = @pm.create_user('u1')
+      @another_u1 = @pm2.create_user('another u1')
```

Comment:
might be convenient for these ivar names to mention what pm they are associated with